features/FxA-I-11/README.md

```diff
@@ -0,0 +1,128 @@
+# Match 1 - Control vs. No Show
+> Eye Password Experiment FXA-80
+
+Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeeley/eye-password-emoji/features/FxA-80-eye-password-experiment/README.md)
+
+Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
+
+## Outcomes
+
+The success of this feature is being able to answer the question:
+
+"Is the 'Show' password button useful in its current form?".
+"Does it reduce the number of errors the users make on the change password page?"
+
+## Assumptions
+
+- Users use this functionality on Android more.
+- We believe the "Show" password button is useful.
+
+## Problem Statement
+
+The change password flow can be improved in the current system.
+
+## Hypothesis
+
+We believe that removing the unmasking functionality from both new and old password fields for people changing their passwords will achieve more errors and bounces.
+
+We will know this is true when we see an increase in bounces to the section, and Task Success errors outlined below.
+
+### Measure
+
+We will build at least **eight** graphs.
+
+The four main graphs are:
+
+1. "Flame chart" for those who do not have an opportunity to use "Show" password.
+2. "Flame chart" for those who can "Show Password"
+3. Change password submission for control
+4. Change password submission for treatment
```

Comment:
It's not immediately obvious how graphs (3) and (4) will be used.  I take it they're the inverse of the bounce rate, so an increase in these graphs corresponds to the hypothesized decrease in bounce rate.  But it'd probably be clearer if we directly build a graph of the bounce rate using e.g:

   `(1 - (sum(settings.change_password.submit) / sum(screen.settings.change_password))) * 100`

And call it "change password bounce rate" so that it's obvious how the graph ties back to the hypothesis.